src/Frontend/Core/Engine/TemplateModifiers.php

```diff
@@ -11,7 +11,6 @@
 
 use Frontend\Core\Engine\Model as FrontendModel;
 use Frontend\Core\Engine\Block\Widget as FrontendBlockWidget;
-use Frontend\Core\Language\Language;
```

Comment:
Wrong implementation, the other one will throw deprecation warnings so you should create an alias for it and use that alias